### ISSUE TITLE
Fix Joint Selector Drop Down Menu

### ISSUE THREAD

**Issue by courtneymcbeth** [parasollab/VR_Robot#28]
I recently found out that some of the prefabs that were depended on by RADER actually lived in the VR_Robot project outside of the RADER package. In moving some UI-related prefabs into the RADER package, the dropdown menu began behaving strangely. It now shows 'Option A', 'Option B', and 'Option C' as well as the joints and I can't seem to make it scroll so I can see beyond the first four items.

Basically, I'd appreciate it if someone could figure out how to remove these extra options and get it to scroll like usual again
